lib/graphics.js

```diff
@@ -57,3 +57,19 @@ export function generateColor(step) {
     var c = "#" + ("00" + (~ ~(r * 255)).toString(16)).slice(-2) + ("00" + (~ ~(g * 255)).toString(16)).slice(-2) + ("00" + (~ ~(b * 255)).toString(16)).slice(-2);
     return (c);
 }
+
+
+/**
+ * gets page width - compatible with different browser types
+ * from https://stackoverflow.com/questions/1038727/how-to-get-browser-width-using-javascript-code
+ * @returns page width
+ */
+export function getWidth() {
```

Comment:
this feels like a bit more of a `lib/user-agent.js` concept (user agent = browser) than a `lib/graphics.js` concept (graphics = drawing stuff)